client/components/navbar/navbar.service.js

```diff
@@ -0,0 +1,476 @@
+                if (!this.otrsPopupService.isLoaded()) {
+                    this.otrsPopupService.init();
+                } else {
+                    this.otrsPopupService.toggle();
+                }
+
+                if (typeof callback === "function") {
+                    callback();
+                }
+            }
+        });
+
+        // Tickets list
+        assistanceMenu.push({
+            title: this.$translate.instant("common_menu_support_list_ticket"),
+            url: "#/support"
+        });
+
+        // Telephony (External)
+        if (_(currentSubsidiaryURLs).has("support_contact")) {
+            assistanceMenu.push({
+                title: this.$translate.instant("common_menu_support_telephony_contact"),
+                url: currentSubsidiaryURLs.support_contact,
+                isExternal: true
+            });
+        }
+
+        return {
+            name: "assistance",
+            title: this.$translate.instant("common_menu_support_assistance"),
+            iconClass: "icon-assistance",
+            subLinks: assistanceMenu
+        };
+    }
+
+    getLanguageMenu () {
+        const currentLanguage = _.find(this.LANGUAGES.available, (val) => val.key === localStorage["univers-selected-language"]);
+
+        return {
+            name: "languages",
+            label: _(currentLanguage).get("name"),
+            title: _(currentLanguage).get("key").replace("_", "-"),
+            subLinks: _.map(this.LANGUAGES.available, (lang) => ({
+                title: lang.name,
+                click: function () {
+                    localStorage["univers-selected-language"] = lang.key;
+                    window.location.reload();
+                },
+                lang: _.chain(lang.key).words().head().value()
+            }))
+        };
+    }
+
+    getUserMenu (currentUser) {
+        return {
+            name: "user",
+            title: currentUser.firstname,
+            iconClass: "icon-user",
+            nichandle: currentUser.nichandle,
+            fullName: `${currentUser.firstname} ${currentUser.name}`,
+            subLinks: [
+                {
+                    title: this.$translate.instant("common_menu_account"),
+                    url: this.REDIRECT_URLS.userInfos
+                }, {
+                    title: this.$translate.instant("common_menu_billing"),
+                    url: this.REDIRECT_URLS.billing
+                }, {
+                    title: this.$translate.instant("common_menu_renew"),
+                    url: this.REDIRECT_URLS.services
+                }, {
+                    title: this.$translate.instant("common_menu_orders_all"),
+                    url: this.REDIRECT_URLS.orders
+                }, {
+                    title: this.$translate.instant("common_menu_contacts"),
+                    url: this.REDIRECT_URLS.contacts
+                }, {
+                    title: this.$translate.instant("common_menu_consumptions"),
```

Comment:
Not needed anymore. You can remove it (see my previous comment)

Reply:
removed